scipy/ndimage/interpolation.py

```diff
@@ -62,12 +63,24 @@ def spline_filter1d(input, order=3, axis=-1, output=numpy.float64):
     output : ndarray or dtype, optional
         The array in which to place the output, or the dtype of the returned
         array. Default is `numpy.float64`.
+    %(mode)s
```

Comment:
How does this work?

Reply:
The actual text is in `ni_docstrings.py`, and this gets magically filled in by the `@_ni_docstrings.docfiller` decorator.